agent/command/archive_auto_create.go

```diff
@@ -0,0 +1,91 @@
+package command
+
+import (
+	"context"
+
+	"github.com/evergreen-ci/evergreen/agent/internal"
+	"github.com/evergreen-ci/evergreen/agent/internal/client"
+	agentutil "github.com/evergreen-ci/evergreen/agent/util"
+	"github.com/evergreen-ci/evergreen/util"
+	"github.com/mholt/archiver/v3"
+	"github.com/mitchellh/mapstructure"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+)
+
+type autoArchiveCreate struct {
+	// Target is the archive file that will be created. The target file's
+	// extension will determine the archiving format to use.
+	Target string `mapstructure:"target" plugin:"expand"`
+
+	// SourceDir is the directory to compress.
+	SourceDir string `mapstructure:"source_dir" plugin:"expand"`
+
+	// Include is a list of filename blobs to include from the source directory,
+	// e.g. "*.tgz", "file.txt", "test_*"
+	Include []string `mapstructure:"include" plugin:"expand"`
```

Comment:
Can we specify that the whole source dir will be used if this is empty in the comment here (I know it's already stated in the docs)

Reply:
Done.